source/dub/dub.d

```diff
@@ -65,6 +65,7 @@ static this()
 
 /// The URL to the official package registry.
 enum defaultRegistryURL = "http://code.dlang.org/";
+enum fallbackRegistryURL = "https://code-mirror.dlang.io/";
```

Comment:
> this week will I hopefully finally get around setting up a stable mirror myself).

GitHub didn't send my review, so I finally got around setting up a stable mirror myself:

-> https://code-mirror2.dlang.io

Heroku Deploy is a bit more difficult to setup due to https://github.com/rejectedsoftware/vibe.d/issues/1785, but I will get around this as well.